.gitignore

```diff
@@ -3,3 +3,4 @@
 tmp/
 output/
 node_modules/
+editions/pluginlibrary/files/
```

Comment:
It might be better to include `tmp` or `output` in the path and thus avoid having to update this file at all.

Reply:
The problem here is, that the following commands will start the nodejs test server. As far as I know, it only serves files from the `files` subdirectory. Or do I miss something? ... It would be simple to use the output dir, if we can server files from there. 

```
cd /your/path/to/tw5/editions/pluginlibrary
tiddlywiki --build test-server
```

Reply:
@Jermolene ... I did create a new route, that lets us serve the test content from the `tmp` folder, relative to the `editions/pluginlibrary` folder. So it will be excluded with the existing .gitignore settings. ... 

But it duplicates the code from the existing $:/core/modules/server/routes/get-file.js file into the pluginlibrary edition. So it will only be available for this edition. 

... I think that approach is a bit hacky. ... I'm not sure about it, if our users start to use it for their own setups. If it's local only I would be OK, but if they expose it to the web ?!

What do you think?